tools/js/test/audit_version_regression.js

```
@@ -0,0 +1,125 @@
+
+        // Track the highest version index seen for each data type
+        const highestVersionIndexForDataType = {}
+
+        // Iterate through versions in chronological order
+        for (const version of versions) {
+          // Skip versions that don't exist in dataPaths
+          if (!platformData[version]) continue
+
+          const versionData = platformData[version]
+
+          for (const [dataType, dataPath] of Object.entries(versionData)) {
+            // Skip non-version data paths (like "latest" or cross-platform refs)
+            const pathParts = dataPath.split('/')
+            if (pathParts.length !== 2) continue
+
+            const [dataPathPlatform, dataPathVersion] = pathParts
+
+            // Skip cross-platform references
+            if (dataPathPlatform !== platform) continue
+
+            // Get the chronological index of the data path version
+            const dataPathIndex = getVersionIndex(platform, dataPathVersion)
+
+            // Skip if version not found in versions.json (like "latest")
+            if (dataPathIndex === null || dataPathIndex === -1) continue
+
+            // Initialize tracking for this data type if not seen before
+            if (!highestVersionIndexForDataType[dataType]) {
+              highestVersionIndexForDataType[dataType] = {
+                version: dataPath,
+                index: dataPathIndex
+              }
+              continue
+            }
+
+            // Check if current data path is older than the highest we've seen
+            const previousHighest = highestVersionIndexForDataType[dataType]
+            if (dataPathIndex < previousHighest.index) {
+              // Skip flagging pre-releases and release candidates for using older data,
+              // as this appears to be intentional behavior (they reuse previous stable data)
+              const isPreRelease = version.includes('-pre') || version.includes('-rc')
```

It's not intentional, let the tests fail if they fail. No special handling for any version 

Data path should be fixed on failure